### PR TITLE
Reagent container visualizer

### LINKED ISSUE #3002: Re-implement spriting for beakers
The sprites were previously handled entirely serverside, which got removed. `SharedSolutionContainerComponent` sets a  `SolutionContainerVisualState` on the `AppearanceComponent`, so just a visualizer is needed.

### PR DESCRIPTION
**Pull request by Macoron**
## About the PR 
Fixes #3002.
Add bluespace beaker and cryostasis beaker (non-obtainable yet)
Also add different variants for chemistry bottles.
Reworked a lot of rsi and meta in chemistry folder.

**Screenshots**

image

**Changelog**

:cl:
- add: Beakers, syringes and chemistry bottles show fill level by different sprites
- add: Add bluespace beaker and cryostasis beaker (non-obtainable yet)

### CODE REVIEW

**Comment by metalgearsloth** [comment]
Will need updating to serv3 when you get time

**Comment by Macoron** [comment]
Done. Though new YAML Linter fails with:
```Error: /Prototypes/Recipes/Crafting/Graphs/tiles.yml(10,15)  Field "icon" not found in "Content.Shared.Construction.MaterialConstructionGraphStep".
Error: /Prototypes/Recipes/Crafting/Graphs/tiles.yml(13,15)  Field "name" not found in "Content.Shared.Construction.MaterialConstructionGraphStep".
Error: /Prototypes/Recipes/Crafting/Graphs/tiles.yml(28,15)  Field "icon" not found in "Content.Shared.Construction.MaterialConstructionGraphStep".
Error: /Prototypes/Recipes/Crafting/Graphs/tiles.yml(31,15)  Field "name" not found in "Content.Shared.Construction.MaterialConstructionGraphStep".
```

But I guess it's not mine.

File: Resources/Prototypes/Entities/Objects/Specific/chemistry-bottles.yml [hunk 1]
```diff
@@ -0,0 +1,93 @@
+- type: entity
+  name: bottle
+  parent: BaseItem
+  id: BaseChemestryEmptyBottle
```
Comment (mirrorcult) [inline Resources/Prototypes/Entities/Objects/Specific/chemistry-bottles.yml:4]: this is misspelled in a lot of places, should be Chemistry

**Review by metalgearsloth — APPROVED** [review]

**Review by PaulRitter — APPROVED** [review]
really nice work, keep it up 👍

**Pull request merged**